### ISSUE TITLE
Add support of Ukrainian (or Cyrillic in general) language for Lilita One and  Comic Shanns fonts

### ISSUE THREAD

**Issue by vasylkorchynskyi** [excalidraw/excalidraw#8571]
Please, add Ukrainian language for following fonts:

- Lilita One
- Comic Shanns

<img width="866" alt="image" src="https://github.com/user-attachments/assets/b017e70d-3c35-4565-bfc4-228fc07bf3b7">
<img width="848" alt="image" src="https://github.com/user-attachments/assets/9cc61510-c72c-48b2-b1a3-ccab78869858">

**Comment by kaguya3222** [comment]
@vasylkorchynskyi It seems that these fonts don't support Ukrainian language, so I suggest to close it with a "won't fix"

**Comment by Mrazator** [comment]
Yeah, unfortunately, none of the fonts supports cyrillic, so we would likely have to find some fallback font which looks similar.

**Comment by kaguya3222** [comment]
@Mrazator What do you think about these font alternatives ? ![Image](https://github.com/user-attachments/assets/0f1b7a65-a485-4a8a-ad58-b9601c84a1fe)
![Image](https://github.com/user-attachments/assets/d9b1708d-8859-49ce-9e6d-69fd68a958d1)

**Comment by ArtemAvramenko** [comment]
For non-Latin characters, a fallback to the default browser font is made. On Windows, this is most often Times New Roman, which is not even a sans font.

So making a fallback to similar fonts installed on the system would be very desirable:

![Image](https://github.com/user-attachments/assets/87774a64-391c-437b-b490-73cbf9d88081)

One could argue that these fonts are not monospaced, but:
1. Times New Roman is too
2. Code is written in Latin, and Cyrillic is most often found in comments, where monospaced fonts are not often required
